packages/backend-for-frontend/src/routers/authorizationRouter.ts

```diff
@@ -71,9 +71,10 @@ const authorizationRouter = (
       const ctx = fromBffAppContext(req.ctx, req.headers);
 
       try {
-        const saml = Buffer.from(req.body.SAMLResponse, "base64").toString();
```

Comment:
Moving this inside the service function after the first log, we usually don't perform any operation in the router